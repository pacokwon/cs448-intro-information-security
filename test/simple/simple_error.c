int main()
{
    int x = 0;

    int z = 10 / x; // Error
}
