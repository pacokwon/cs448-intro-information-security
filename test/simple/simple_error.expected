Divide-by-zero detected at line 5 and col 16
